AppLovin MAX Demo App - ObjC/AppLovin MAX Demo App - ObjC/ALAppDelegate.m

```diff
@@ -7,6 +7,7 @@
 //
 
 #import "ALAppDelegate.h"
+#import "Adjust.h"
```

Comment:
Since Adjust is a framework, you _should_ be able to import it as `#import <Adjust/Adjust.h>`.